src/autopas/LogicHandler.h

```diff
@@ -962,20 +962,19 @@ class LogicHandler {
 
   /**
    * Helper Method for computeRemainderInteractions3B. This method calculates all interactions between one buffer
-   * particle and two particles from the container.
+   * particle and two particles from the container. It first creates neighbor lists for all buffer particles in parallel
```

Comment:
I feel a little reluctant to call these "neighbor lists", to avoid confusion with our verlet lists i.e. with a skin and kept for several timesteps. Maybe see if you can think of a better name - I don't have any good ideas. Otherwise, I think it would also be fine to just briefly explain the difference to Verlet lists.